database_refresh/02-cronjob.yaml

```diff
@@ -0,0 +1,75 @@
+---
+    apiVersion: batch/v1beta1
+    kind: CronJob
+    metadata:
+      name: db-refresh-job
+    spec:
+      schedule: "30 10 * * 0"
```

Comment:
10:30 on a Sunday

Reply:
err. yes... that's the same as the other jobs

Reply:
Sorry, wasn't anything wrong with the cron, I was just echoing it back when I'd parsed it. Ignore me! 🙈